doc/OnlineDocs/src/expr/managing.py

```diff
@@ -33,35 +33,6 @@
 print(EXPR.expression_to_string(e, labeler=NumericLabeler('x')))
 # @ex2
 
-# ---------------------------------------------
```

Comment:
Just curious - why remove this example?

Reply:
- ex3: the example won't run: `standardize=` has not been a valid argument for 6 years.
- ex4: `clone_counter` is deprecated (and nonfunctional), so we don't want to continue to advertise it.

Reply:
"Our docs are up to date, we swear."